templates/index.html

```diff
@@ -16,54 +16,56 @@ <h1>Universal Operator Lifecycle Manager</h1>
 				</p>
 			</div>
 			<div class="col-7 u-align--center u-vertically-center">
-				<ul class="p-inline-images u-no-margin--bottom">
-					<li class="p-inline-images__item">
-						<a href="#install">
-							{{
-								image(
-									url="https://assets.ubuntu.com/v1/57d86853-linux-logo-white.svg",
-									alt="Linux",
-									height="88",
-									width="75",
-									hi_def=True,
-									loading="eager",
-									attrs={"class": "p-inline-images__logo"},
-								) | safe
-							}}
-						</a>
-					</li>
-					<li class="p-inline-images__item">
-						<a href="#install">
-							{{
-								image(
-									url="https://assets.ubuntu.com/v1/82c19e2e-macos-logo-white.svg",
-									alt="macOS",
-									height="30",
-									width="127",
-									hi_def=True,
-									loading="eager",
-									attrs={"class": "p-inline-images__logo"},
-								) | safe
-							}}
-						</a>
-					</li>
-					<li class="p-inline-images__item">
-						<a href="#install">
-							{{
-								image(
-									url="https://assets.ubuntu.com/v1/bfc851de-windows-logo-white.svg",
-									alt="macOS",
-									height="33",
-									width="144",
-									hi_def=True,
-									loading="eager",
-									attrs={"class": "p-inline-images__logo"},
-								) | safe
-							}}
-						</a>
-					</li>
-				</ul>
-			</div>
+        <div class="hero-image">
+          <ul class="p-inline-images u-no-margin--bottom">
```

Comment:
Instead of adding new styling. You could just remove this no-margin-bottom and add some extra with the spacing class.
```html
<ul class="p-inline-images u-sv3">
```